.github/files/generate-ci-matrix.php

```diff
@@ -58,7 +58,7 @@
 		'script'  => 'test-php',
 		'php'     => $php,
 		'wp'      => 'latest',
-		'timeout' => 15, // 2021-01-18: Successful runs seem to take ~8 minutes for PHP 5.6 and for the 7.4 master run, ~5.5-6 for 7.x and 8.0.
+		'timeout' => 20, // 2022-01-25: 5.6 tests have started timing out at 15 minutes. Previously: Successful runs seem to take ~8 minutes for PHP 5.6 and for the 7.4 master run, ~5.5-6 for 7.x and 8.0.
```

Comment:
FYI, the cause is 1300 new Search/Sync tests added in #19674 that added about 5 minutes to the Jetpack PHPUnit tests. See also p1643047778112500-slack-CBG1CP4EN